lib/Sema/TypeChecker.cpp

```diff
@@ -400,6 +400,13 @@ void swift::performTypeChecking(SourceFile &SF, TopLevelContext &TLC,
     if (Options.contains(TypeCheckingFlags::ForImmediateMode))
       TC.setInImmediateMode(true);
 
+    if (Options.contains(TypeCheckingFlags::SkipNonInlinableFunctionBodies))
+      // Disable this optimization if we're compiling SwiftOnoneSupport, because
```

Comment:
We shouldn't be generating a swiftinterface for SwiftOnoneSupport, though?

Reply:
This also applies to -emit-module